apps/article/stylesheets/index.styl

```diff
@@ -57,4 +57,32 @@
     garamond-size('s-headline')
   &__explore
     height 45px
-    margin auto
\ No newline at end of file
+    margin auto
+
+.articles-featured-articles
+  margin gutter-size auto
+  padding 20px 0
+  border-bottom 1px solid gray-lighter-color
+  h4
+    avant-garde()
```

Comment:
Ah yes, these styles I copied over from the `articles` subapp. Some older style choices remain.